sdk/storage/inc/blobs/internal/protocol/blob_rest_client.hpp

```diff
@@ -0,0 +1,2630 @@
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <functional>
+#include <map>
+#include <stdexcept>
+#include <string>
+#include <type_traits>
+#include <vector>
+
+namespace libXML2 {
+#include "libxml/tree.h"
+}
+
+#include "context.hpp"
+#include "http/http.hpp"
+#include "http/pipeline.hpp"
+
+namespace Azure { namespace Storage { namespace Blobs {
```

Comment:
>Blobs [](start = 48, length = 5)

Since protocol layer is bound to a specific REST API version, I would suggest this is defined here or at least commented here. Otherwise there is no way for customer to know that.

Reply:
Most of the APIs (if not all of them) can run even if there's no `x-ms-version` header. I will add this header in the future.

Reply:
added